test.sh

```diff
@@ -102,6 +102,10 @@ vagrantUp() {
   cd ${WORKSPACE}/RackHD/example
   cp -rf ${WORKSPACE}/build-config/vagrant/* .
   CONFIG_DIR=${CONFIG_PATH} WORKSPACE=${WORKSPACE} REPO_NAME=${REPO_NAME} vagrant up --provision
+  if [ $? == 1 ]; then 
```

Comment:
Since the vagrant up can fail due to a lot reason, so it's better change to `$? -ne 0`